src/components/molecules/Alert/Alert.test.tsx

```diff
@@ -2,56 +2,34 @@ import { composeStories } from '@storybook/react';
 import { render, screen } from 'src/test-utils.js';
 import * as stories from './Alert.stories';
 
-const { Default, WithCloseButton } = composeStories(stories);
+const { Info, Error } = composeStories(stories);
 
 describe('Alert Component', () => {
-    it('should render a Alert', () => {
-        render(<Default />);
+    it('should render an Alert', () => {
+        render(<Info />);
     });
 
-    it('should render solid variant if variant is not provided', () => {
-        render(<Default />);
+    it('should render info variant if variant is not provided', () => {
```

Comment:
incorrect test statement.
<Info /> already has severity: AlertSeverity.Info defined.

Reply:
Changing it to remove AlertSeverity.Info in args and renamed story from Info to Default